### ISSUE TITLE
i don't know why my model is not load in my python script..

### ISSUE THREAD

**Issue by kushwah15** [kushwah15/Blood-detection#4]

**Comment by AmanSinghal08** [comment]
from tensorflow.keras.models import load_model
from tensorflow.keras.preprocessing import image
import numpy as np
import cv2
model = load_model('model2.h5')
test_image = image.load_img(r"Datasets\train\Normal\img12.png", target_size = (224,224))
test_image = image.img_to_array(test_image)
test_image=test_image/255
test_image = np.expand_dims(test_image,axis=0)
result = model.predict(test_image)
print(result)

//this code will help you (try it)
